src/app/modules/project/components/carousel.vue

```diff
@@ -1,44 +1,83 @@
 <template>
-    <div class="">
-    	<div class="container -fluid -nowrap pad-4-v">
-    		<div class="col" v-for="item in content.items">
-    			<prismic-image :field="item.image" class="" />
-	    	</div>
-    	</div>
+    <div class="carousel">
+		<div class="carousel-wrapper">
+			<p ref="status" class="carousel-status"></p>
+			<div ref="carousel" class="main-carousel" >
+				<div class="carousel-cell" v-for="item in content.items">
+					<prismic-image :field="item.image" class="carousel-image" />
+				</div>
+			</div>
+		</div>
 	</div>
 </template>
 
 ***********************************************************
 
 <script>
 
+import Flickity from 'flickity';
+
 export default
 {
-	"components": {},
 
 	///////////////////////////////////////////////////////
 	// 	...
 	///////////////////////////////////////////////////////
 
 	data ()
 	{
-		return {}
+		return {
+			"flkty" : null,
+		}
 	},
 	"props" : ["content"],
-	"watch" : {},
+	"watch" : 
+	{
+		// Recreate flickity when content changes
+		content ( value ) { this.createFlickity() },
+	},
 
 	///////////////////////////////////////////////////////
 	// 	...
 	///////////////////////////////////////////////////////
 
-	"mounted": function(){},
+	"mounted": function()
+	{
+		this.createFlickity()
+	},
+
 	"destroyed": function(){},
 
 	///////////////////////////////////////////////////////
 	// 	...
 	///////////////////////////////////////////////////////
 
-	"methods" : {},
+	"methods" : 
+	{
+		createFlickity ()
+		{
+			// Init flickity
+			var isMobile = matchMedia('screen and (max-width: 768px)').matches;
+			let options =
+			{
+				contain: false,
+				pageDots: false,
+				imagesLoaded: false,
```

Comment:
@antonlegoo shouldn't `imagesLoaded` be `true`?

Reply:
doh, can't believe I missed that one. Really good catch! That should now be fixed.

And I also double-backed on the [documentation for flickity-imagesloaded](https://github.com/metafizzy/flickity-imagesloaded) which confused me at first but now I think I'm using it correctly. In order to use set `imagesLoaded: true`, you need to import the `flickity-imagesloaded` package and init() with that instead of the regular `flickity`. 🤷‍♂️ 

This also brought to light an issue with component reuse. If you navigate from one project to another project, and the same exact set of components are used within both, then Vue will not destroy the component.. it will reuse it. So if you nav'ed to slide 2 on a carousel component in project 1, then go to project 2 that carousel component will stay on slide 2! That should now be fixed.

Reply:
Dope! Works great. That makes sense about initializing with the `flickity-imagesloaded` package.

Oh yeah I actually noticed the Video component doesn't always re-render the content inside. I can make a ticket for that.

Looks like there's a small merge conflict, but after let's merge this in  🚀